.github/workflows/main.yml

```diff
@@ -133,6 +134,7 @@ jobs:
       - name: Jackson compatibility tests / 2.6.4 for Spark 2.4.4
         uses: gradle/gradle-build-action@v2
         with:
+          cache-read-only: true
```

Comment:
seems unintentional that all other jackson versions below still write to the cache?

Reply:
Only the first usage of the gradle-action pulls the cache

Reply:
ok, adding it would be redundant but maybe also more consistent?